src/Elliptic/Systems/Elasticity/BoundaryConditions/LaserBeam.hpp

```diff
@@ -0,0 +1,160 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
```

Comment:
you only use `magnitude` in the cpp file

Reply:
Unfortunately this file defines the `Tags::Magnitude` :(

Reply:
Oh right... :(